constraint-solver/src/reachability.rs

```diff
@@ -23,9 +23,10 @@ where
 
 /// Returns the set of all variables reachable from an initial set via shared constraints
 /// (algebraic constraints and bus interactions).
-/// The set of blocking variables is a barrier that stops the reachability search.
-/// The returned set does not contain the blocking variables but it does contain
-/// the initial variables.
+/// The set of blocking variables is a barrier that stops the reachability search, in the
```

Comment:
The definition of 'blocking variable' did not change, I just clarified a little what it means.